x-pack/plugins/security/public/authentication/access_notice/access_notice_page.tsx

```diff
@@ -67,33 +67,37 @@ export function AccessNoticePage({ http, fatalErrors, notifications }: Props) {
       }
     >
       <form onSubmit={onAcknowledge}>
-        <div className="euiModal secAccessNoticePage__container">
-          <div className="euiModal__flex">
-            <div className="euiModalBody euiModalBody__overflow secAccessNoticePage__text">
-              <div className="euiModalBody__overflow secAccessNoticePage__text">
+        <EuiPanel paddingSize="none" className="secAccessNoticePage__container">
+          <EuiFlexGroup gutterSize="none" direction="column">
+            <EuiFlexItem className="secAccessNoticePage__textWrapper">
+              <div className="secAccessNoticePage__text">
                 <EuiText textAlign="left">
                   <ReactMarkdown>{accessNotice}</ReactMarkdown>
                 </EuiText>
               </div>
-            </div>
-            <div className="euiModalFooter secAccessNoticePage__footer">
-              <EuiButton
-                fill
-                type="submit"
-                color="primary"
-                onClick={onAcknowledge}
-                isDisabled={isLoading}
-                isLoading={isLoading}
-                data-test-subj="accessNoticeAcknowledge"
-              >
-                <FormattedMessage
-                  id="xpack.security.accessNotice.acknowledgeButtonText"
-                  defaultMessage="Acknowledge and continue"
-                />
-              </EuiButton>
-            </div>
-          </div>
-        </div>
+            </EuiFlexItem>
+            <EuiFlexItem className="secAccessNoticePage__footer">
+              <EuiFlexGroup gutterSize="none">
+                <EuiFlexItem grow={false}>
```

Comment:
Looks like you're trying to get out of the `display: flex` here. You can do this more simply by just using a simple `div`.
```suggestion
              <div>
```